start_bs.py

```diff
@@ -169,9 +169,9 @@ class SampleXYZ(Device):
     from mxtools.raster_flyer import MXRasterFlyer
     raster_flyer = MXRasterFlyer(vector, zebra, detector)
 
-    from nyxtools.robot import DensoOphydRobot
+    from nyxtools.isara_robot import IsaraRobotDevice
     from denso_robot import DensoRobot
-    denso_ophyd_robot = DensoOphydRobot("XF:19IDC-ES{Rbt:1}", name="robot")
+    denso_ophyd_robot = IsaraRobotDevice("XF19IDC-ES{Rbt:1}", name="robot")
```

Comment:
if you can make an Isara or generic subclass of DensoRobot, then you will also be able to clean up this and the following line with appropriate names for the objects and classes

Reply:
I will make a more generic name for it.

Reply:
added a more generic interface with 2b38bd0